docs/migrating.rst

```diff
@@ -19,50 +20,63 @@ replacement for the html5lib one.
 
 Here is an example of replacing the sanitization method:
 
-.. doctest::
+.. code::
```

Comment:
In order for these to be doctest blocks and pass our tests, we'd have to install html5lib as a dev requirement and I don't want to do that because I think it'll make it harder to maintain everything if html5lib is both a dev requirement and also a vendored package.

I don't want to use the vendored version because it makes these illustrative examples harder to understand.

Further, these examples are missing bits, so we'd have to flesh them out as complete examples.

Given all that, I thought it'd be easier to just turn them into code blocks for now.